build/init-docker.sh

```diff
@@ -20,7 +20,9 @@ if [[ ! $(type -P "$DOCKER") ]]; then
 fi
 
 # Verify docker is reachable.
+set +e
 OUT=$(($DOCKER images > /dev/null) 2>&1)
+set -e
 if [[ ! $? == 0 ]]; then
```

Comment:
I think you can do something like

```bash
OUT=$(($DOCKER images  > /dev/null) 2>&1) || (echo "Docker is not reachable. Did you follow the installation instructions?"; echo "'docker images': ${OUT}")
```

to avoid that hack.